susApp/www/js/DrawerLinks.js

```diff
@@ -0,0 +1,51 @@
+angular.module('DrawerLinks', [])
+
+.controller('DrawerController', function($state, $scope, $location, $ionicLoading, $window, $ionicHistory, $timeout, $localStorage) {
```

Comment:
Will be god to add some documentation.

E.g

/**
 * @file
 */

/**
 * [A description]
 * @param {type} [name]
 */